### ISSUE TITLE
veh gives option to stay on same spell instead of offering the next one

### ISSUE THREAD

**Issue by jeremygurr** [jeremygurr/dcssca#245]

**Comment by G0rg0** [comment]
Yes please!

**Comment by Sandman25DCSS** [comment]
I think it should not be changed for nightmare mode.

**Comment by Sandman25DCSS** [comment]
Basically any item/god/whatever buff should not happen on nightmare if possible.

**Comment by jeremygurr** [comment]
I'm not sure that this is a straight buff. It could mean getting 1 or 2 spells earlier than you normally would have, but I can't see it really impacting late game, since by then you've usually come across most spells just in spellbooks found or in stores. But delaying veh from switching spells also increases the time it takes for the more powerful spells that veh offers to be presented, which counter balances the buff. At best this is a very minor buff. 

Having too many obscure / hard to document things different between the difficulty versions creates a spoilery atmosphere that I'm trying to get away from. Ideally the list of differences between any two difficulty modes could be summarized in a short paragraph. 

For example, not only do hard mode players not get the uninterruptible stair climb for abyss stairs like the other difficulty modes, but did you know that even ascending any stairs as a hard mode player, you receive a much higher evasion malus than other difficulty levels? That's an obscure spoilery thing that someone switching modes is going to be surprised by in a bad way. Nightmare should be significantly more difficult, but not because it has more surprises that the player has no way of knowing or predicting, but because of simple to understand and clearly published changes to gameplay variables / mechanics.  

So if there is a buff that you'd really not want to have in nightmare, then maybe it should be balanced with a nerf, or not included at all, for all difficulty levels. 

I do understand your fear of power creep weakening the experience of nightmare mode. That can be countered easily enough though through other means I think. It's possible that with the rune curses in effect you could use a few more buffs being added to nightmare mode anyway :)

I don't believe this kind of buffing is the same as a straight power creep / increase monster difficulty cycle duvessa was talking about. I think the difference is in the nature of the buff. For this case, I think it makes veh a little more pleasant, without really impacting the overall balance of the game. Yeah overall it can be considered a buff, but a minor one at best. And I'm not only buffing here. Adding movement penalty and stamina limitations, as well as stair dance removal are significant nerfs (proven by the fact that I had to water down tomb:3 to compensate).

I just nerfed magic pool size a bit too to bring it closer to vanilla crawl, because after a few games as a magic user I realized it was a bit much. 

But to your point I will carefully consider buffs in the context of nightmare mode, and if possible try and find a balancing nerf for it the buff is really necessary.

**Comment by Sandman25DCSS** [comment]
Ok, probably we don't need to do an exception for nightmare mode then. I was trying to keep nightmare mode players rush training spellcasting like it often happens with default Veh.
By the way, probably it can be good to remove stairdancing EV-penalty for all modes, it's a very spoilery thing. I noticed it first when my crawl crashed and I saw EV stats updated on screen during using stairs. Normally it happens too fast and is basically invisible.
